Framework/API/inc/MantidAPI/AlgorithmFactory.h

```diff
@@ -133,6 +133,9 @@ class MANTID_API_DLL AlgorithmFactoryImpl final
   /// Get the algorithm categories
   const std::map<std::string, bool> getCategoriesWithState() const;
 
+  /// Returns a single algorithm descriptor
+  int getAlgLatestVersion(const std::string &algName) const;
```

Comment:
Quick question. What does the `highestVersion` method that already exists do differently?